module-1/dist/assets/js/admin-page.js

```diff
@@ -53,13 +53,21 @@ replyButtons.forEach(elem => {
                 subject: `re: ${message}`,
                 text: text.value 
             }
-            postData(`${hostUrl}sendEmail`, data)
+            postData(`${HOST_URL}sendEmail`, data)
             .then(data => { 
                 // TODO process data
             })
             .catch((error) => {
                 throw new Error(error)
             })
+            form.reset()
+            form[0].value=""
+            form[0].placeholder = "Your mail has been sent"
+            form[0].classList.add('color__placeholder')
+            function formClose() {
+                form.style.display = "none"
+            }
+            setTimeout(formClose, 3000)
```

Comment:
Why do we need a timeout here? Timeouts should be avoided at any costs except very rare cases when they're absolutely necessary

Reply:
I'll take a note. Hovever, to solve your task about demonstration of an approval after a mail exchange, we have come up only with that solution. Could you demonstrate some alternatives to us during today's lesson?